synchronization/src/main/java/de/cyface/synchronization/DatabaseException.java

```diff
@@ -0,0 +1,26 @@
+package de.cyface.synchronization;
+
+/**
+ * An {@code Exception} thrown when a database operation failed.
+ *
+ * @author Armin Schnabel
+ * @version 1.0.0
+ * @since 2.1.0
+ */
+public class DatabaseException extends Exception {
+
+    /**
+     * @param detailedMessage A more detailed message explaining the context for this <code>Exception</code>.
+     */
+    public DatabaseException(final String detailedMessage) {
+        this(detailedMessage, null);
```

Comment:
There should be a super constructor taking only one argument, so that you can avoid calling this with null.

Same is true for all the other exception types.